### PR TITLE
Bug 1243115 revival

### PR DESCRIPTION
**Pull request by jotes**
Hi, I decided to help and provide some small refactorings and solve problems with combined filters.

### CODE REVIEW

**Comment by mathjazz** [comment]
Just checking - what's the status of this?

**Comment by mathjazz** [comment]
/cc @BychekRU

Let me know if you need any help.

**Comment by jotes** [comment]
@mathjazz I need to fix the tests and write new test-cases for the fix. I plan to work on it tomorrow.

**Comment by jotes** [comment]
Ok, it looks like that filters weren't applied correctly: https://docs.djangoproject.com/en/dev/topics/db/queries/#spanning-multi-valued-relationships . I'll try to apply a solution in a moment.

File: pontoon/base/utils.py [hunk 1]
```diff
@@ -563,8 +563,7 @@ def parse_time_interval(interval):
     def parse_timestamp(timestamp):
         return timezone.make_aware(datetime.strptime(timestamp, '%Y%m%d%H%M'), timezone=pytz.UTC)
 
-    start = interval.split('-')[0]
-    end = interval.split('-')[1]
+    start, end = interval.split('-')
```
Comment (mathjazz) [inline pontoon/base/utils.py]: Beautiful!

File: pontoon/base/models.py [hunk 2]
```diff
@@ -1180,6 +1181,24 @@ class EntityQuerySet(models.QuerySet):
     """
     Queryset provides a set of additional methods that should allow us to filter entities.
     """
+    user_entity_filters = ('missing', 'fuzzy', 'suggested', 'translated')
```
Comment (mathjazz) [inline pontoon/base/models.py]: Maybe `user_status_filters`?

Comment (jotes) [inline pontoon/base/models.py]: done :)

**Pull request merged**